Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -350,7 +352,12 @@ public override void OnPositionInputChanged(InputEventData<Vector2> eventData)
                 {
                     canTeleport = false;
                     teleportEnabled = false;
-                    TeleportSystem.RaiseTeleportStarted(this, TeleportHotSpot);
+
+                    if (TeleportSurfaceResult == TeleportSurfaceResult.Valid ||
```

Comment:
I see distance being calculated but it is not being considered here... is that intentional? is distance a factor in whether or not a teleport should be allowed?

Reply:
At this point we've already calculated a valid point. So I'm not sure what you mean here.

Reply:
was just wondering if distance was involved in determining a valid point.

Reply:
It is not